src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -1,16 +1,57 @@
 package com.google.codeu.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+// Handles fetching and saving user data.
 @WebServlet("/about")
 public class AboutMeServlet extends HttpServlet {
+
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
+
+  // Responds with the "about me" section for a particular user.
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Responds with a hard-coded message for testing purposes.
-    response.getOutputStream().println("hello world");
+
+    response.setContentType("text/html");
+
+    String user = request.getParameter("user");
+
+    if (user == null || user.equals("")) {
```

Comment:
Use `user.isEmpty()` instead of `user.equals("")`. It is more clear to read "If user is null or user is empty" rather than "if user is null or user is equal to empty character".